src/block/tile/Hopper.php

```diff
@@ -35,6 +35,10 @@ class Hopper extends Spawnable implements Container, Nameable{
 
 	private const TAG_TRANSFER_COOLDOWN = "TransferCooldown";
 
+	public const DEFAULT_TRANSFER_COOLDOWN = 10;
```

Comment:
Where did you get that number from, the original pr contained the correct value:
A hopper waits for 4 redstone ticks = 8 game ticks, according to the wiki